src/version.ts

```diff
@@ -6,4 +6,4 @@
  *
  * hardcoded for now, we would want to have it injected dynamically
  */
-export const VERSION = "0.0.8";
```

Comment:
This may be confusing with the other docs, were we going to go forward with 2.0.0, or we need to change the docs instead of saying go to 2.0.0, instead go to the new repository.